css/index.css

```diff
@@ -56,6 +56,84 @@ html, body {
     font-family: 'Roboto', sans-serif;
 }
 
+/* header nav {
+    display: flex;
+    margin: 5px;
+    justify-content: space-evenly;
+    align-content: space-between;
+    align-items: baseline;
+} */
+
+header, nav {
+    width: 40%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px 2%;
+    font-size: 14px;
+    margin: 30px;
+}
```

Comment:
you will want to remove the extra margin on the nav and its a tags to remove that large gap of white space in the top of your index page.